nvflare/lighter/provision.py

```diff
@@ -63,7 +63,6 @@ def handle_provision(args):
     current_path = os.getcwd()
     custom_folder_path = os.path.join(current_path, args.custom_folder)
     sys.path.append(custom_folder_path)
-    print("\nPath list (sys.path) for python codes loading: {} \n".format(sys.path))
```

Comment:
why remove this line?
although not sure if this is needed anymore @IsaacYangSLA can you comment on this